server/api/observation/observation-router.ts

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import getObservationTypeController from './type/observation-type-get.controller';
+import deleteObservationTypeController from './type/observation-type-delete.controller';
+import createObservationTypeController from './type/observation-type-create.controller';
+import getObservationController from './observation-get.controller';
+import createObservationController from './observation-create.controller';
+
+const observationRouter = express.Router() as any;
+
+/* Observation */
+observationRouter.get('/', getObservationController.getAllObservations);
+observationRouter.post('/', bodyParser.json(), createObservationController.createObservation);
+
+/* Observation Type */
+observationRouter.get('/type', getObservationTypeController.getAllObservationTypes);
+observationRouter.delete('/type/:type', deleteObservationTypeController.deleteObservationType);
+observationRouter.post('/type/:type', createObservationTypeController.createObservationType);
```

Comment:
You've obviously called this out already in your 'next steps' but we should probably get the security elements in before approving/merging this PR.

Reply:
Subtasks in Asana where created.

Reply:
Absolutely, I agree. This PR is for merging to my feature branch so it would be safe to merge all these changes before writing more code.